### ISSUE TITLE
The performance is not very good

### ISSUE THREAD

**Issue by clearisoft** [MasayukiSuda/GPUVideo-android#34]
The performance is not very good when the mp4 is 4k and the android hardware is low.
I hope when preview with no filter, the performance run as well as origin

**Comment by pablogeek** [comment]
I agree, it struggles in some slow devices. Have you found any solution? @clearisoft
